src/Components/__Test__/Details.test.js

```diff
@@ -0,0 +1,15 @@
+import { Provider } from 'react-redux';
+import Details from '../Pages/Details';
+import configureStore from '../Redux/ConfigureStore';
+import renderer from 'react-test-renderer';
+
+it('Main renders correctly', () => {
+  const mainSite = renderer
+    .create(
+      <Provider store={configureStore}>
+        <Details />
+      </Provider>
```

Comment:
- [ ] Your tests are failing, also please add more tests in order to test your app more in depth
![image](https://user-images.githubusercontent.com/62705947/206502007-a8698300-67a8-48f4-b2de-856435051b54.png)